themes/classic/_dev/css/components/brands.scss

```diff
@@ -0,0 +1,61 @@
+#manufacturer {
+    #main {
+        ul {
+            display: flex;
+            align-items: flex-start;
+            flex-wrap: wrap;
+
+            .brand {
+                padding: 10px;
+                margin: 5px;
+                text-align: center;
+                background: white;
+                transition: .4s ease-out;
+                width: calc(20% - 10px);
+                text-align: center;
+                min-width: 210px;
+                position: relative;
+
+                &-infos {
+                    a {
+                        &::before {
+                            content: '';
+                            position: absolute;
+                            width: 100%;
+                            height: 100%;
+                            z-index: 1;
+                            top: 0;
+                            left: 0;
+                        }
+                    }
+                }
+
+                &:hover {
+                    box-shadow: 0 0 8px 0 rgba(0, 0, 0, 0.2);
+                }
+
+                &-products {
+                    display: none;
+                }
+
+                &-img {
+                    width: 200px;
+                    margin: auto;
+                }
+            }
+        }
+    }
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
could we use the boostrap variables for 768px? https://getbootstrap.com/docs/4.0/layout/grid/#sass-mixins